tests/unit/test_session.py

```diff
@@ -875,6 +875,23 @@ def test_endpoint_from_production_variants_with_accelerator_type(sagemaker_sessi
         Tags=tags)
 
 
+def test_update_endpoint_succeed(sagemaker_session):
+    ims = sagemaker_session
+    ims.sagemaker_client.describe_endpoint = Mock(return_value={'EndpointStatus': 'InService'})
+    endpoint_name = "some-endpoint"
+    endpoint_config = "some-endpoint-config"
+    returned_endpoint_name = sagemaker_session.update_endpoint(endpoint_name, endpoint_config)
+    assert returned_endpoint_name == endpoint_name
+
+
+def test_update_endpoint_fail(sagemaker_session):
```

Comment:
let's call this test 'test_update_endpoint_existing_endpoint'

Reply:
Spoke with Chuyang offline, my mistake. Let's name this update_endpoint_non_existing_endpoint.